boteval/templates/admin/mturk/_macros.html

```diff
@@ -19,6 +20,27 @@
 <p class="alert alert-warning"> (Approval is not applicable)</p>
 {% endif %}
 
+<script>
+    function notify_user() {
+        document.getElementById("user_note").innerHTML = "You have successfully given the user a bonus payment! If you click on the bonus button again, you'll give the user an additional bonus payment.";
+    }
+</script>
+
+{% if asgn['AssignmentStatus'] == 'Approved' %}
+<section id = bonus_button>
+
+<p>Base pay given to worker for completing task: {{base_pay}} </p>
+<p>Hourly rate of pay desired for worker: {{pay_per_hour}} dollars/hour</p>
+<p>Bonus to be given: ${{bonus_pay}}</p>
+<form method="POST" action="{{url_for('admin.' + crowd_name + '_give_bonus', worker_id=asgn['WorkerId'], payment=bonus_pay, asgn_id=asgn['AssignmentId'])}}" onsubmit="notify_user()">
+    <button type="submit" value="Submit" class="col-3 mb-2 btn btn-success btn-sm">Give Bonus </button>
+    <span class="col-1"></span>
+</form>
+<p>Note: You can change the base pay and hourly rate of pay in this assignment's configuration file.</p>
```

Comment:
is that actually possible? i think you'd have to take down and bring up the boteval for that to happen and it would apply to all users. A good future step would let this get modified for a particular instance by editing a text field before clicking. but for now i wouldn't even mention this.